src/mpi_setup.f90

```diff
@@ -104,27 +104,17 @@ subroutine mpi_setup_mol2(ierr)
     include 'mpif.h'
 
     call Broadcast(quick_molspec,ierr)
-!    call MPI_BARRIER(MPI_COMM_WORLD,mpierror)
 
     call MPI_BCAST(dcoeff,nbasis*maxcontract,mpi_double_precision,0,MPI_COMM_WORLD,mpierror)
     if (quick_method%ecp) then
       call MPI_BCAST(eta,nprim,mpi_double_precision,0,MPI_COMM_WORLD,mpierror)
     endif
 
-! DFT Parameter
-    if (quick_method%DFT.or.quick_method%SEDFT) then  
-      call MPI_BCAST(quick_method%nof_functionals,1,mpi_integer,0,MPI_COMM_WORLD,mpierror)
-      call MPI_BCAST(quick_method%functional_id,size(quick_method%functional_id),mpi_integer,0,MPI_COMM_WORLD,mpierror)
-      call MPI_BCAST(quick_method%xc_polarization,1,mpi_double_precision,0,MPI_COMM_WORLD,mpierror)
```

Comment:
This was deleted - is the subroutine where xc_polarization is set called by all MPI tasks? I was trying to follow the code but got lost... I guess it must be set otherwise the open shell tests would fail.

Reply:
I have added broadcasting this variable in quick_method module.